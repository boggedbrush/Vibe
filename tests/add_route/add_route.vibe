# VibeSpec: 1.6
patch_type: add_block
file: hello.py
position: before
anchor: "^# Entry point"
--- code: |
    @app.route('/timestamp')
    def get_timestamp():
        from datetime import datetime
        return jsonify({'timestamp': datetime.now().isoformat()})

