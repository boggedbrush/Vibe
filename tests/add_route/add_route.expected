# hello.py
def greet(name):
    print(f"Hello, {name}!")


class Greeter:
    def __init__(self, name):
        print(f'Hello {name}')


@app.route('/timestamp')
def get_timestamp():
    from datetime import datetime
    return jsonify({'timestamp': datetime.now().isoformat()})


# Entry point
if __name__ == '__main__':
    print('here')
